sdk/core/Azure.Core/src/AccessToken.cs

```diff
@@ -19,6 +19,7 @@ public AccessToken(string accessToken, DateTimeOffset expiresOn)
         {
             Token = accessToken;
             ExpiresOn = expiresOn;
+            TokenType = "Bearer";
```

Comment:
nit: should these be constants, if not an extensible enum?

Reply:
discussed [here](https://github.com/Azure/azure-sdk-for-net/pull/45134#discussion_r1722201101)